README.adoc

```diff
@@ -1164,6 +1164,32 @@ result = if some_condition; something else something_else end
 result = some_condition ? something : something_else
 ----
 
+=== Use `case` [[use-case]]
```

Comment:
We should come up with a better name for the title and the anchor. Something like `case vs if/else` or `When to use case?`. Many of the headings and anchors are really bad today for historical reasons - titles didn't exist until recently and were auto-generated from anchors which admittedly could have been much better.

Reply:
Changed.